modules/auxiliary/server/http_proxy.rb

```diff
@@ -0,0 +1,147 @@
+require 'msf/core/exploit/http/proxy'
+
+class MetasploitModule < Msf::Auxiliary
+
+  include Msf::Auxiliary::Report
+  include Msf::Exploit::Remote::HttpServer::Proxy
+
+  def initialize
+    super(
+      'Name'        => 'HTTP Proxy',
+      'Description' => %q{
+        This module creates an HTTP proxy using Rex' HTTP modules.
```

Comment:
```suggestion
        This module creates an HTTP proxy using Rex's HTTP modules.
```

Reply:
Far as I was taught, words ending in a phonetic "s" lack the suffix letter. I'll defer to the English language experts (ping proper English speakers, at least @timwr and @OJ)

Reply:
The use of phonetics as a means to determine apostrophe placement to denote ownership is new to me. I was under the impression that the rule applied only to words ending with `s`, regardless of phonetics.

While I would hardly consider `grammarbook.com` a definitive resource, it was the first Google result, and makes no mention of phonetics.

* https://www.grammarbook.com/punctuation/apostro.asp

However, proper nouns introduce additional complexity:

> Rule 1b. Many common nouns end in the letter `s` (lens, cactus, bus, etc.). So do a lot of proper nouns (Mr. Jones, Texas, Christmas). There are conflicting policies and theories about how to show possession when writing such nouns. There is no right answer; the best advice is to choose a formula and stay consistent.

None the less, I reject both your suggestions, in favor of removing the apostrophe entirely:

```suggestion
        This module creates an HTTP proxy using Rex HTTP modules.
```

Both `Rex` and `HTTP` in combination as `Rex HTTP` could be interpreted, correctly, as A Thing, without any requirement to show ownership/possession. In the same way that I write Microsoft Office documents using Microsoft Office, and audit in line with CIS benchmarks.

Despite the phonetic `s` of `Office`, I have only ever written a Microsoft Office document. I have never written a Microsoft Office' document, nor a Microsoft Office's document. Similarly, I have never audited in line with CIS's benchmarks, nor CIS' benchmarks, only CIS benchmarks.

Are we done here?